met-api/src/met_api/models/engagement.py

```diff
@@ -41,11 +42,12 @@ def get_all_engagements(cls):
         return engagements_schema.dump(query)
 
     @classmethod
-    def save_engagement(cls, engagement) -> DefaultMethodResult:
-        """Save engagements."""
+    def create_engagement(cls, engagement) -> DefaultMethodResult:
+        """Save engagement."""
         new_engagement = Engagement(
             name=engagement["name"],
             description=engagement['description'],
+            rich_text_state=engagement['rich_text_state'],
```

Comment:
get is somehow safer and better than bracket indexing
so engagement.get('rich_text_state',None) will be the better usage